doc/whats_new/v1.5.rst

```diff
@@ -139,6 +139,11 @@ more details.
   transformers' ``fit`` and ``fit_transform``. :pr:`28205` by :user:`Stefanie
   Senger <StefanieSenger>`.
 
+- |Feature| :class:`ensemble.StackingClassifier` and
```

Comment:
I think this might need to move to 1.6 now.

Reply:
I've moved it :)